feature/qos/otg_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go

```diff
@@ -1215,6 +1223,15 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	q := d.GetOrCreateQos()
 	queues := netutil.CommonTrafficQueues(t, dut)
 
+	if dut.Vendor() == ondatra.NOKIA {
```

Comment:
Similar as my comment in https://github.com/openconfig/featureprofiles/pull/2024/files#r1326530972

Reply:
Added new deviation `qos_queue_requires_id`.